src/coreclr/src/jit/lower.cpp

```diff
@@ -3231,38 +3231,39 @@ void Lowering::LowerStoreLocCommon(GenTreeLclVarCommon* lclStore)
 //
 void Lowering::LowerRetStruct(GenTreeUnOp* ret)
 {
-#if defined(FEATURE_HFA) && defined(TARGET_ARM64)
```

Comment:
Why remove the `#if defined(TARGET_ARM64)`? You've enabled this code for arm32. Is that correct? Maybe you can get away with it because we don't support SIMD on arm32.

Reply:
I did not intend to make this change. Thank you for catching it.

---
In reply to: [510499387](https://github.com/dotnet/runtime/pull/43706#discussion_r510499387) [](ancestors = 510499387)